dhall/src/Dhall/Normalize.hs

```diff
@@ -916,7 +922,7 @@ isNormalized e0 = loop (Syntax.denote e0)
 True
 >>> "x" `freeIn` "y"
 False
->>> "x" `freeIn` Lam "x" (Const Type) "x"
+>>> "x" `freeIn` Lam (mfb "x" (Const Type)) "x"
```

Comment:
`mfb` is pretty far away here and it's not even obvious why it should be in scope.

```suggestion
>>> "x" `freeIn` Lam (makeFunctionBinding "x" (Const Type)) "x"
```